### PR TITLE
Allow file names with multiple dots in certs:add command

### PR DESCRIPTION
**Pull request by raphaklaus**
Certificate files like mydomain.com.crt and mydomain.com.key will be better interpreted.

### CODE REVIEW

**Comment by josegonzalez** [comment]
Can you update the tests for this? Just so we don't accidentally break it later.

**Comment by raphaklaus** [comment]
Okay, added this scenario too.

File: tests/unit/10_certs.bats [hunk 1]
```diff
@@ -33,6 +33,13 @@ teardown() {
   assert_success
 }
 
+@test "(certs) certs:add with multiple dots in the filename" {
+  run bash -c "dokku certs:add $TEST_APP $BATS_TMPDIR/tls/domain.com.crt $BATS_TMPDIR/tls/domain.com.key"
```
Comment (josegonzalez) [inline tests/unit/10_certs.bats]: Haven't checked, but these files do exist, right?

Comment (raphaklaus) [inline tests/unit/10_certs.bats]: Where does files like server.crt and server.key are in this repository, GitHub search sucks :\

Comment (josegonzalez) [inline tests/unit/10_certs.bats]: They are from [here](https://github.com/raphaklaus/dokku/tree/b0777a44cb7e1a4d8d30934798f95cd500a9d7ba/tests/unit) and the tar file is extracted [here](https://github.com/dokku/dokku/pull/2622/files#diff-3d75297c33bf18c21085b512d18b3d7dR5).

Comment (raphaklaus) [inline tests/unit/10_certs.bats]: Done. But something broke the tests... Not sure if is something I did. Will check.

Comment (raphaklaus) [inline tests/unit/10_certs.bats]: Okay, it's done @josegonzalez

**Pull request merged**